proto/crypto/types.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package tendermint.proto.crypto;
+
+option go_package = "github.com/tendermint/tendermint/proto/crypto";
+
+import "third_party/proto/gogoproto/gogo.proto";
+
+option (gogoproto.equal_all)    = true;
+option (gogoproto.compare_all)  = true;
+option (gogoproto.populate_all) = true;
+
+message PublicKey {
```

Comment:
This will evolve in the future and will be compatible with the other Proto work you're doing correct?

Reply:
this is all that is needed on the tendermint side. Since we only support ed25519 keys for tendermint and the SDK will define their own proto key types.